site/profile/manifests/core/telegraf.pp

```diff
@@ -0,0 +1,60 @@
+# @summary
+#   Monitor system metrics.
+#
+# @param password
+#   The InfluxDB password for the telegraf user
+#
+# @param username
+#   The InfluxDB username for node metrics (as opposed to summit power metrics).
+#
+# @param database
+#   The InfluxDB database for node metrics (as opposed to summit power metrics).
+#
+# @param host
+#   The backing InfluxDB instance. At present the default DNS record is a CNAME.
+class profile::core::telegraf(
+  String $password,
+  String $username = 'telegraf-nodes',
+  String $database = 'nodes',
+  String $host     = 'metrics-ingress.ls.lsst.org',
+) {
+  class { '::telegraf':
+    hostname    => $::facts['fqdn'],
+    global_tags => {'site' => $::site},
+  }
+
+  $default_inputs = {
+    'cpu'        => [{}],
+    'chrony'     => [{}],
```

Comment:
It might be reasonable to slice this out to a `profile::core::chrony` module, but it's harmless as-is.